src/test/test_set.cpp

```diff
@@ -23,7 +23,7 @@ TestReport InterpreterTestSet::execute() {
 			ExitStatusTag answer = Interpreter::execute(m_source, m_dump, f);
 
 			if (ExitStatusTag::Ok != answer)
-				return {TestStatusTag::Fail};
+				return { .m_code = TestStatusTag::Fail, .m_msg = "Error in: " + m_source };
```

Comment:
We are trying to keep the codebase as standard C++14. If you could change this line accordingly, that'd be great